runtime/runtime/src/gas_keys.rs

```diff
@@ -409,4 +413,50 @@ mod tests {
             .count();
         assert_eq!(gas_key_count, 0);
     }
+
+    #[test]
+    fn test_view_gas_keys() {
```

Comment:
Could you please separate this test case into 3 separate ones: testing view_gas_key output, testing gas key does not exist with view_gas_key, testing view_gas_keys?

 More granular and smaller tests and easier to understand and debug.

Reply:
Could you also please add a test testing view_gas_keys without any gas keys present?